src/commands/Slots.js

```diff
@@ -0,0 +1,137 @@
+const { Command } = require('@dyno.gg/dyno-core');
+
+class Slots extends Command {
+    constructor(...args) {
+        super(...args);
+
+        this.aliases      = ['slots'];
+        this.module       = 'Fun';
+        this.description  = 'Spin the slots and win the jackpot! ...or loose everything.';
+        this.usage        = 'slots';
+        this.example      = 'slots';
+        this.cooldown     = 5000;
+        this.expectedArgs = 0;
+    }
+	checkJackpot(gameid, bonus) {

```

Comment:
Constructor uses 4 spaces, the rest of the code is using tabs, can you convert the constructor idention to tabs?